src/utils/api/account/lsk.js

```diff
@@ -79,15 +80,21 @@ export const getAccount = async ({
       params: normParams,
     });
 
-    return response.data[0];
+    if (response.data[0]) {
+      const account = { ...response.data[0] };
+      const isAccountUninitialized = !account.summary?.publicKey;
```

Comment:
Since you assumed `response.data[0]` is an object, then the account existed and `account.summary` is an object too. You don't need the null check.